examples/app_commands/slash_cog.py

```diff
@@ -0,0 +1,18 @@
+from discord.ext import commands
+from discord.commands import slash_command # Importing the decorator that makes slash commands.
+
+class Example(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+ 
+    @slash_command(guild_ids=[...]) # Create a slash command for the supplied guilds.
+    async def hello(self, ctx):
+        await ctx.respond("Hi, this is a slash command from a cog!")
+    
+
+    @slash_command() # Not passing in guild_ids creates a global slash command (might take an hour to register).
```

Comment:
Tell the users that this is a global command

Reply:
`creates a global slash command` it's already there wdym

Reply:
oops